.eslintrc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
```

Comment:
rename to eslintrc unless you are planning to use any js evaluations
also add ignore into this file rather than having a separate one

Reply:
done